specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,128 @@
+                                // Allow the browser to process F7 key
+                                CHECK_FAILURE(args2->put_AllowBrowserHandle(TRUE));
+                            }
+                        }
+                        return S_OK;
+                    })
+                    .Get(),
+                &m_acceleratorKeyPressedToken));
+
+        }
+    }
+```
+
+## C#
+```c#
+    CoreWebView2Settings _webViewSettings = webView.CoreWebView2.Settings;
+    // All browser accelerator keys are disabled with this setting
+    _webViewSettings.AreBrowserAcceleratorKeysEnabled = false;
+
+    webView.CoreWebView2.AcceleratorKeyPressed += WebView2Controller_AcceleratorKeyPressed;
+
+    void WebView2Controller_AcceleratorKeyPressed(object sender, CoreWebView2AcceleratorKeyPressedEventArgs e)
+    {
+        switch (e.KeyEventKind)
+        {
+            case CoreWebView2KeyEventKind.KeyDown:
+            case CoreWebView2KeyEventKind.SystemKeyDown:
+                {
+                    Keys keyData = (Keys)(e.VirtualKey);
+                    // Allow the browser to process F7 key
+                    if (e.VirtualKey == Key.F7) 
+                    {
+                        e.AllowBrowserHandle = true;
+                    }
+                    break;
+                }
+            
+        }
+    }
+
+```
+
+# Remarks
+By default, `AreBrowserAcceleratorKeyEnabled` is `TRUE` and default value for `AllowBrowserHandle` is `TRUE`. When developers set `AreBrowserAcceleratorKeyEnabled` to `FALSE` this will also change the default value for `AllowBrowserHandle` to `FALSE`. If developers want specific keys to be handled by the browser after changing the `AreBrowserAcceleratorKeyEnabled` settings to `FALSE`, they need to enable them by setting `event_args->put_ AllowBrowserHandle(TRUE)`. This option will give the event arg higher priority over the settings when the browser decide whether to handle the keys.  
+
+# API Details
+## C++
+```
+/// This is a continuation of the ICoreWebView2AcceleratorKeyPressedEventArgs interface.
+[uuid(45238725-3774-4cfe-931f-7985a1b5866f), object, pointer_default(unique)]
+interface ICoreWebView2AcceleratorKeyPressedEventArgs2 : ICoreWebView2AcceleratorKeyPressedEventArgs {
+  /// These APIs allow devs to enable/disable the browser from handling specific accelerator keys.
+  /// By default, `AreBrowserAcceleratorKeyEnabled` is `TRUE` and `AllowBrowserHandle` is `TRUE`.
+  /// When devs set `AreBrowserAcceleratorKeyEnabled` to `FALSE` this will override and the
+  /// default value for `AllowBrowserHandle` will be `FALSE` and prevent the browser to handle
+  /// accelerator keys. If users want specific keys to be handled by the browser after changing
+  /// the settings to `FALSE`, they need to enable them by setting `AllowBrowserHandle` to `TRUE`.
+  /// This option will give the event arg higher priority over the `AreBrowserAcceleratorKeyEnabled`
+  /// settings when we handle the keys.

```

Comment:
Also, on the topic of the Handled property please also update the ref docs for it. It doesn't make it clear what exactly is getting marked handled or not (of the various things that might happen from accelerator key press)